apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.ts

```diff
@@ -22,6 +28,7 @@ export class MembershipToggleComponent implements OnInit {
   @Input() shapes: NbComponentShape = 'rectangle';
   @Input() status: NbComponentStatus = 'basic';
   @Input() size: NbComponentSize = 'small';
+  @Input() appearance: NbButtonAppearance = 'filled';
```

Comment:
Let's define a common enum for different sizes somewhere in the models and call it everywhere.